packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1346,11 +1346,12 @@ export default class Meetings extends WebexPlugin {
 
   /**
    * syncs all the meeting from server
+   * @param {boolean} keepOnlyLocusMeetings - whether the sync should keep only locus meetings or any other meeting in meetingCollection
    * @returns {undefined}
    * @public
    * @memberof Meetings
    */
-  public syncMeetings() {
+  public syncMeetings(keepOnlyLocusMeetings = true): undefined {
```

Comment:
nitpick: The name of the parameter might be clearer as `removeNonLocusMeetings`, but not too worried if you think `keepOnlyLocusMeetings` is clearer.

Reply:
sdk team is not so happy with the term 'non-locus meetings' which is why I prefer thisname